examples/seed-data-storage/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "@nhost-examples/seed-data-storage",
```

We need to tell the package is private so our CD won't try to publish it
```suggestion
  "name": "@nhost-examples/seed-data-storage",
  "private": true,
```